website/content/docs/plugins/install-plugins.mdx

```diff
@@ -22,50 +19,81 @@ command, and those requirements are explicitly documented in the template.
 be repeated for as many plugins as you need.
 We recommend this for JSON as the template cannot contain the information about the required plugins.
 
-Finally, you can manually install any plugin. This is mostly useful if you're in an environment with
-restricted internet access, or if you're installing non-final versions of plugins.
+While `packer plugins install` is able to install plugins remotely, you can use
+the `--path` flag to install locally-sourced plugin binaries.
+This is typically useful in two general cases:
+
+1. you are in an environment that does not have access to the internet
+1. you are trying to install a development version of a plugin for testing.
 
 Refer to the [Installation Guides](#installation-guides) section of this page for information about
-each, including usage examples.
+each method, including usage examples.
 
 The remainder of this document will serve as documentation on how Packer interacts with plugins.
 We encourage you to read this to get familiar with this process, as it will help you troubleshoot
-your builds if you encounter problems with that.
+your builds if you encounter problems with plugin discovery/loading.
```

Comment:
lets reword this section a bit to be a bit more succinct and remove the use of discovery/loading. 
```
The remainder of this document documents how Packer interacts with plugins.

We encourage you to read this to get familiar with this process, as it will help you troubleshoot builds if you encounter problems with using external plugins.
```

Reply:
Yeah it's a decent reword, I'll apply that.
Only thing I'll maybe drop is the "external" plugin, since plugins are by definition external and we don't embed plugins anymore